pytest_bdd/gherkin_terminal_reporter.py

```diff
@@ -58,7 +58,7 @@ def pytest_runtest_logstart(self, nodeid, location):
 
     def pytest_runtest_logreport(self, report):
         rep = report
-        res = self.config.hook.pytest_report_teststatus(report=rep)
+        res = self.config.hook.pytest_report_teststatus(report=rep, config=self.config)
```

Comment:
`output_must_be_the_same_as_regular_reporter` check started to fail due to below warning on pytest 4.2.0:
```
test_gherkin.py::test_scenario_1
  /Users/milosz.sliwinski/workspace/pytest-bdd/.tox/py27-pytestlatest/lib/python2.7/site-packages/pytest_bdd/gherkin_terminal_reporter.py:61: UserWarning: Argument(s) ('config',) which are declared in the hookspec can not be found in this hook call
    res = self.config.hook.pytest_report_teststatus(report=rep)
```
Indeed [pytest_report_teststatus hook](https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_report_teststatus) takes `config` as an argument so I just passed it and warning disappeared.